Mathlib/CategoryTheory/Shift/Pullback.lean

```diff
@@ -103,4 +118,49 @@ lemma pullbackShiftFunctorAdd'_hom_app :
     ← Functor.map_comp, Iso.hom_inv_id_app, Functor.map_id]
   rfl
 
+namespace CommShift
+
+variable {D : Type*} [Category D] [HasShift D B] (F : C ⥤ D) [F.CommShift B]
+
+open Functor in
+/-- If `F : C ⥤ D` commutes with the shifts on `C` and `D`, then it also commutes with
+their pullbacks by an additive map.
+-/
+@[simps]
+noncomputable def Pullback.CommShift :
```

Comment:
This should also be in the `Functor.CommShift` namespace, maybe as `Functor.CommShift.pullback`.

Reply:
If `commShiftOp` is in the `Functor` namespace, would'nt it make sense to call this `Functor.commShiftPullback`?

Reply:
Otherwise I'd want to call the other`Functor.CommShift.op` (and `Functor.CommShift.unop`), for uniformity. But then if you open the `Functor.CommShift` namespace, then Lean might get confused about the meaning of `op`.

Reply:
Yes, `commShiftPullback` is good.